addon/helpers/build-secondary-nav-links.js

```diff
@@ -102,11 +102,15 @@ export default Ember.Helper.extend({  // Helper defined using a class, so can in
             );
         }
 
-        if (this.get('currentUser.user.canViewReviews')) {
-            links.PREPRINTS.insertAt(1, {
-                name: 'eosf.navbar.reviews',
-                href: serviceLinks.reviewsHome,
-            });
+        if (session.get('isAuthenticated')) {
+            this.get('currentUser.user').then((user) => {
```

Comment:
If `links.PREPRINTS` is returned and used before this promise resolves, will the My Reviewing link still be added?

Reply:
Yup!